### PR TITLE
Clamp-pipe_CU-knavr6

### PR DESCRIPTION
**Pull request by a-wilkes**

### CODE REVIEW

**Comment by a-wilkes** [comment]
Task linked: [CU-knavr6 Clamp pipe](https://app.clickup.com/t/knavr6)

**Comment by lpboucher** [comment]
Task linked: [CU-knavr6 Clamp pipe](https://app.clickup.com/t/knavr6)

File: src/app/shared/components/grid-list/grid-list.component.ts [hunk 1]
```diff
@@ -10,11 +11,14 @@ export class GridListComponent implements OnInit {
     @Input() itemColumnSpan: number = 1;
     @Input() gridGap: number = 0;
 
-    constructor() { }
+    constructor(private clamp: ClampPipe) { }
```
Comment (lpboucher) [inline src/app/shared/components/grid-list/grid-list.component.ts]: variable name should be clampPipe, to better represent what it is

File: src/app/shared/components/grid-list/grid-list.component.ts [hunk 2]
```diff
@@ -10,11 +11,14 @@ export class GridListComponent implements OnInit {
...
 
     ngOnInit(): void {
-        this.mobileColumnCount = this.clampColumnValue(this.mobileColumnCount);
-        this.itemColumnSpan = this.clampColumnValue(this.itemColumnSpan);
+        const MAX_COLUMN_COUNT = 12;
```
Comment (lpboucher) [inline src/app/shared/components/grid-list/grid-list.component.ts]: const variable should be defined as below @Input at the top of the class, or event potentially outside the class

**Review by lpboucher — CHANGES_REQUESTED** [review]
Please rebase onto develop before making corrections

**Review by lpboucher — APPROVED** [review]

**Pull request merged**